client/src/components/Header/style.js

```diff
@@ -0,0 +1,26 @@
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  bottom: {
+    backgroundColor: '#24292E',
+    left: 0,
+    right: 0,
+    bottom: 0,
+  },
+  container: {
+    margin: 15,
+    display: 'flex',
+    width: 200,
```

Comment:
Set the width like this `width: '100%'` to make it take the whole screen width for all different kinds of phone screens.